docs/src/content/docs/commands/HGETALL.md

```diff
@@ -57,37 +66,36 @@ HGETALL user:1000
 ### Example 2: Retrieving from a non-existing key
 
 ```DiceDB
-HGETALL user:2000
+127.0.0.1:7379> HGETALL user:2000
 ```
 
 `Output:`
 
 ```
-(empty array)
+(empty list or set)
 ```
 
 ### Example 3: Error when key is not a hash
 
 ```DiceDB
-SET user:3000 "This is a string"
-HGETALL user:3000
+127.0.0.1:7379> SET user:3000 "This is a string"
+OK
+127.0.0.1:7379> HGETALL user:3000
 ```
 
 `Output:`
 
 ```
-(error) WRONGTYPE Operation against a key holding the wrong kind of value
+(error) ERROR WRONGTYPE Operation against a key holding the wrong kind of value
 ```
 
-## Notes
-
-- The `HGETALL` command is a blocking command and can be slow if the hash contains a large number of fields.
-- It is recommended to use this command with caution in production environments where performance is critical, especially if the hash is expected to grow large.
-
-## Best Practices
+### Example 4: Error when an invalid number of arguments are passed
```

Comment:
Instead of Example 4, can you please add title as Error Example and subtitle as `invalid number of arguments are passed`